src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -362,6 +362,103 @@
             }
         {% endif %}
 
+        var twigDebugEnabled = function () {
+            return !!document.getElementById('__twig-debug-bar');
+        };
+
+        var twigDebugEnable = function () {
```

Comment:
the logic in this method is not compatible with old browsers, which is not good for something running in in the toolbar (as the toolbar runs in the userland page, which might need to support them).

so there is 2 solutions here:

- ensure the code is compatible with old IE versions
- feature-detect the support for features being used, and forbid enabling the Twig debug mode if not supported.